src/composables/homeRoute.ts

```diff
@@ -0,0 +1,27 @@
+import { computed, ref, watch } from '@vue/composition-api';
```

Comment:
I really don't like the design of this composable. The `homeRouteName` should not be an reactive thing. This should be a computed prop based on the environment (isLoggedIn / route / store state). But to be honest I would vote for removing this composable and maybe replacing it with a utility function `getHomePageRouteName(route, store)` or something similar.